cfme/utils/appliance/implementations/ui.py

```diff
@@ -90,9 +90,11 @@ def ensure_page_safe(self, timeout='10s'):
         # THIS ONE SHOULD ALWAYS USE JAVASCRIPT ONLY, NO OTHER SELENIUM INTERACTION
 
         def _check():
-            result = self.browser.execute_script(self.ENSURE_PAGE_SAFE, silent=True)
```

Comment:
Can we get this added to cfme/utils/appliance/implementations/ui.py in the ENSURE_PAGE_SAFE function

Reply:
I have to check appliance version in JS then. Could you advice how I could check appliance version?

Reply:
@izapolsk you can just check if the thing is undefined, remember that adding more javascript is basically "free" for us. So you would do `if(typeof window.ManageIQ.gtl.loading !== "undefined"){...}`. Or something like that. If we keep it as it is now in the PR, we will get a small slowdown.